src/js/renderers/flash.js

```diff
@@ -237,6 +237,7 @@ const FlashMediaElementRenderer = {
 			}
 			;
 		methods.push('stop');
+		methods.push('canplay');
```

Comment:
Or in that case you will need to send this inside the `window['__ready__${flash.id}]` after the `rendererready` event like this:

```javascript
let event = createEvent('canplay', flash);
mediaElement.dispatchEvent(event);
```